trieste/acquisition/utils.py

```diff
@@ -101,3 +101,15 @@ def split_optimizer(
         return optimizer(search_space, (taf, n) if isinstance(f, tuple) else taf)
 
     return split_optimizer
+
+
+def select_first_output(x: TensorType) -> TensorType:
+    """
+    A utility function for trajectory sampler-related acquisition functions which selects the first
+    output as the trajectory to be used.
+
+    :param x: Input with shape [..., B, L], where L is the number of outputs of the model.
+    :return: TensorType with shape [..., B], where the first output has been selected to reduce the
+        input.
+    """
+    return x[..., 0]
```

Comment:
I wonder if we might want this function to be slightly more general, Like if we actually wanna use multiple latents, then we need to be able to access the samples of each one separately? Maybe make a select_nth_output function ?

Reply:
Also this could do with some (very simple) unit tests.